src/app/components/pages/requests/forms/request/request.component.html

```diff
@@ -432,7 +432,7 @@
       <mat-form-field [appearance]="formFieldsStyle" [class.invalid-field]="conferenceFeePaymentTypeSelect.value == null && validationFailed">
```

Comment:
I believe you can use onChange on mat-form-field component, it could be nicer solution